src/performance/dashboard/src/components/modals/__tests__/ModalModifyLoadTests.test.js

```diff
@@ -54,10 +54,43 @@ describe('<ModalModifyLoadTests />', () => {
     });
 
     test('modal has a valid heading', () => {
-        render(wrapper)
+        render(wrapper);
         expect(document.querySelector('.ModalModifyLoadTest .bx--modal-header__heading').innerHTML).toBe('Load test configuration');
     });
 
+    describe('Tests inputs on method dropdown field', () => {
+        test('the modal dialog has a method dropdown field', () => {
+            const { container } = render(wrapper);
+            const selectedItem = container.querySelectorAll('#method .bx--list-box__field span');
+            expect(selectedItem.length).toBe(1);
+        });
+
+        test('the dropdown has exactly 2 selectable menu choices', () => {
+            const { container } = render(wrapper);
+            const dropDown = container.querySelector('#method .bx--list-box__field');
+            fireEvent.click(dropDown);
+            expect(document.querySelectorAll('#method  .bx--list-box__menu-item__option').length).toBe(2);
+        });
+
+        test('selecting GET and POST update state', () => {
```

Comment:
Could this message maybe describe what is being tested here? i.e `selecting GET and POST correctly updates state` ?

Reply:
Updated message text